src/Request/CustomFields/Create/Request.php

```diff
@@ -44,7 +41,16 @@ public function customField()
         return $this->customField;
     }
 
-    protected function endpoint()
+    /**
+     * @return Response
+     */
+    public function send()
```

Comment:
Návrhuji toto:

do AbstractRequest dáme @template TResponse a v abstractRequest budeme vracet TResponse takže nemusíme dělat tyto overridy :)

Reply:
Jj presne tak sem to chtel udelat